### ISSUE TITLE
Use case for wine64 vs winewow?

### ISSUE THREAD

**Issue by pikajude** [NixOS/nixpkgs#21913]
I spent fruitless hours this morning trying to install Steam in my NixOS using wine64 before eventually realizing that I needed to install winewow instead. During the course of my debugging, irc.freenode.net #winehq told me that pure wine64 is not a supported configuration and should not be expected to work.

Is there a use case for pure 64-bit wine over winewow? Should both be packaged in nixpkgs?

**Comment by tohl** [comment]
> Should both be packaged in nixpkgs?

They are (from wine/default.nix):

build = "wineWow"; # "wine32", "wine64", "wineWow"

**Comment by pikajude** [comment]
> They are (from wine/default.nix):

That's my question. Should they be?

**Comment by spy-tech** [comment]
64 bits wine is used by people who want to run Windows applications which require a lot of RAM. Games generally don't hit those levels. 

Options are good, but I am surprised you need to know that wine even exists as a Steam user on NixOS. 

Valve should just maintain a Nix expression :)

**Comment by pikajude** [comment]
But wineWow can run 64 bit apps, right? Is 64-bit only wine necessary here?

**Comment by bendlas** [comment]
WineWOW needs the wine64 binary.

**Comment by gleber** [comment]
Based on the comments, it looks like there `winewow` depends on wine64 binaries, so the question seem to be answered. I think the issue can be closed.

**Comment by c0bw3b** [comment]
Issue has been clarified.
Thanks to everyone. :)

**Comment by bendlas** [comment]
> WineWOW needs the wine64 binary.

I commented this a bit in a hurry. While the statement is not wrong, it's not a good reason for `wine64` being called exactly that, or even for it to be in `all-packages.nix`.
To be quite honest, I'm not quite happy with wine configuration in nix either, but if we were to change the naming, it should be done once, coherently and the current method should be deprecated, not removed. I'll open a separate ticket for that.
